frontend/src/component/admin/roles/RoleForm/useRoleForm.ts

```diff
@@ -79,7 +57,7 @@ export const useRoleForm = (
     });
 
     const isNameUnique = (name: string) => {
-        return !roles.some(
+        return ![...roles, ...projectRoles].some(
```

Comment:
Girlscouting: Only root roles were being checked for uniqueness. This now checks both.